Model/Ui/AdyenHppConfigProvider.php

```diff
@@ -114,19 +121,23 @@ public function getConfig()
         $dob = "";
 
         // get customer
-        if ($this->_customerSession->isLoggedIn()) {
+        if ($this->customerSession->isLoggedIn()) {
             $gender = \Adyen\Payment\Model\Gender::getAdyenGenderFromMagentoGender(
-                $this->_customerSession->getCustomerData()->getGender()
+                $this->customerSession->getCustomerData()->getGender()
             );
 
             // format to calendar date
-            $dob = $this->_customerSession->getCustomerData()->getDob();
+            $dob = $this->customerSession->getCustomerData()->getDob();
             if ($dob) {
                 $dob = strtotime($dob);
                 $dob = date('m/d/Y', $dob);
             }
         }
 
+		$config['payment']['adyenHpp']['originKey'] = $this->adyenHelper->getOriginKeyForBaseUrl();
```

Comment:
I don't think origin key is needed here. Can you please double check?

Reply:
Checked it, indeed is working without them. Going to remove them.

Reply:
originKey is only for secured fields/card component indeed